.github/workflows/welcome_message.yml

```
@@ -0,0 +1,18 @@
+name: 'Welcome New Contributors'
+
+on:
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  welcome-new-contributor:
+    runs-on: ubuntu-latest
+    steps:
```

I recommend adding [a `timeout-minutes` configuration](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes) for **every** Actions job to ensure you don't end up wasting the default 6 hours in the case of a runaway Action/job:

```suggestion
    timeout-minutes: 2
    steps:
```